secure_mail/management/commands/email_signing_key.py

```diff
@@ -101,8 +104,10 @@ def handle_label(self, label, **options):
         output = ''
 
         if options.get('print_private_key'):
-            output += gpg.export_keys([self.key.fingerprint], True)
-
+            try:
+                output += gpg.export_keys([self.key.fingerprint], True, passphrase=SIGNING_KEY_DATA["Passphrase"])
+            except:
+                raise CommandError("SIGNING_KEY_DATA needs to have a Passphrase asigned")
```

Comment:
```suggestion
                raise CommandError("SIGNING_KEY_DATA needs to have a passphrase value assigned.")
```

Typo here. And I think it is better to raise a warning here instead of a `CommandError`, since a passphrase is only required on GnuPG version 2 and later, and some users may still be using GnuPG 1.x, which does not require a passphrase to export keys. I would like the warning text to include this tidbit, and a link to the appropriate section of the python-gnupg documentation: https://pythonhosted.org/python-gnupg/#deployment-requirements